external-crates/move/crates/move-binary-format/src/normalized.rs

```diff
@@ -118,12 +118,12 @@ pub struct Constant<S> {
     pub data: Vec<u8>,
 }
 
-/// Normalized version of a `StructDefinition`. Not safe to compare without an associated
-/// `ModuleId` or `Module`.
+/// Normalized version of a `StructDefinition`.
 #[cfg_attr(test, derive(Clone))]
 #[derive(Debug)]
 pub struct Struct<S: Hash + Eq> {
     pub name: S,
+    pub datatype: Datatype<S>,
```

Comment:
I don't understand what this is. What is the type here? Itself? If so, what are the type arguments?